evmrpc/server.go

```diff
@@ -0,0 +1,57 @@
+package evmrpc
+
+import (
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+type EVMServer interface {
+	Start() error
+}
+
+func NewEVMRpcServer(
```

Comment:
might be better to call this `NewEVMHttpServer` since they both use RPC

Reply:
Also, not sure if we'll need this but seems like it would be hard to make an HTTPServer with both HTTP and WS enabled on the same server since each one creates a new HTTP server.

Reply:
updated